config_check/refresh.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# requires  autoconf, automake and libtool
+# See https://github.com/wolfSSL/wolfssl/blob/master/INSTALL
+
+# we expect to be starting in the config_check directory, so move to the parent
+cd ".."
+
+# the location of wolfSSL where the ./configure script should run
+WOLFSSL_REPO="$PWD"
+
+
+if [ ! -f "configure" ]; then
+    echo "configure not found! did you forget to run autogen.sh in $PWD?"
+    exit 1
+fi
+
+echo This WOLFSSL_REPO = $PWD
+
+# the directory where output files go (a github repo is helpful for tracking changes)
+WOLFSSL_FILE_ROOT="$WOLFSSL_REPO/config_check"
+echo "WOLFSSL_FILE_ROOT = $WOLFSSL_FILE_ROOT"
+
+mkdir -p "$WOLFSSL_FILE_ROOT"
+
+# set a variable for the input command
+WOLFSSL_CMD_FILE="$WOLFSSL_FILE_ROOT/cmd.txt"
+
+# setup some variables for output files
+WOLFSSL_OUTPUT="$WOLFSSL_FILE_ROOT/output.txt"
+WOLFSSL_OPTIONS="$WOLFSSL_FILE_ROOT/options.h"
+WOLFSSL_YES="$WOLFSSL_FILE_ROOT/Enabled-Features.txt"
+WOLFSSL_NO="$WOLFSSL_FILE_ROOT/Disabled-Features.txt"
+
+# we'll want to run configure from the root directory of wolfssl
+cd "$WOLFSSL_REPO"
+
+# save current help text for reference
+./configure --help > "./help.txt"
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    echo "Error"
+    exit $retVal
+fi
+
+
+# show the command text found
+echo "CMD File= $WOLFSSL_CMD_FILE"
+echo ""
+
+# get the contents of the command file, trimming all text after the # character
+WOLFSSL_CMD="$(cat $WOLFSSL_CMD_FILE | cut -d'#' -f1)"
```

Comment:
Is error checking important here? when I comment out `./configure` in cmd.txt it doesn't work but it can't really do any damage and just leads to my terminal telling me command not found followed by all the text below

Reply:
I added some error checking and comments in the `cmd.txt` file. There's not much difference other than not doing the file operations after the `./configure` command.